container_main.py

```
@@ -66,20 +66,28 @@ def run_scheduler():
+            f"0.0.0.0:{port}",
+            "--workers",
+            "2",
+            "--timeout",
+            "120",
+            "--access-logfile",
+            "-",
+            "--error-logfile",
+            "-",
+            "api_server:app",
+        ]
+        subprocess.run(cmd, check=True, env={**os.environ, "FLASK_DEBUG": "false"})
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)